python/ray/train/tests/conftest.py

```diff
@@ -21,17 +21,6 @@ def ray_start_4_cpus():
     ray.shutdown()
 
 
-@pytest.fixture
-def ray_start_runtime_env():
-    # Requires at least torch 1.11 to pass
-    # TODO update torch version in requirements instead
-    runtime_env = {"pip": ["torch==1.11.0"]}
```

Comment:
Do you know why this was added in the first place and why it's not longer needed?

Reply:
it's because the torch version we had in requirements was too old. should be good now